tests/integration/deploy/test_deploy_command.py

```diff
@@ -423,6 +423,40 @@ def test_deploy_without_s3_bucket(self, template_file):
             deploy_process_execute.stderr,
         )
 
+    @parameterized.expand(["aws-serverless-function.yaml", "cdk_v1_synthesized_template_zip_functions.json"])
```

Comment:
We can probably just test against a single template since its an expected failure with a CLI option. The action here is to simply remove the `@parameterized.expand`, removing the `template_file` in the arguments, and changing `template_path = self.test_data_path.joinpath(template_file)` to just use `aws-serverless-function.yaml` instead.

This way, we only end up using that single template to validate the new command line argument.

Reply:
sure